core-drawer-panel.html

```diff
@@ -75,344 +75,425 @@
 @homepage github.io
 -->
 
+<link rel="import" href="../polymer/polymer.html">
+
 <link rel="import" href="../core-media-query/core-media-query.html">
 <link rel="import" href="../core-selector/core-selector.html">
 
-<polymer-element name="core-drawer-panel" touch-action="auto">
-<template>
-
+<dom-module id="core-drawer-panel">
   <link rel="stylesheet" href="core-drawer-panel.css">
 
-  <core-media-query query="max-width: {{forceNarrow ? '' : responsiveWidth}}" queryMatches="{{queryMatches}}"></core-media-query>
+  <template>
+    <core-media-query query="[[mediaQuery]]" on-query-matches-changed="onQueryMatchesChanged"></core-media-query>
+
+    <core-selector class-name="[[coreSelectorClass]]" attrForSelected="id" selected="{{selected}}">
 
-  <core-selector class="{{ {'narrow-layout' : narrow, transition : transition, dragging : dragging, 'right-drawer': rightDrawer} | tokenList }}" valueattr="id" selected="{{selected}}">
+      <div id="main" style="[[mainStyle]]">
+        <content select="[main]"></content>
+        <div id="scrim" on-tap="[[togglePanel]]"></div>
```

Comment:
I'd like to add `activateEvent=""` to the `core-selector` here to prevent its behavior entirely and make the `onClick` event on the scrim actually do the work. @frankiefu, what do you think of that?